components/automate-cli/pkg/verifyserver/server/api/v1/s3config_test.go

```diff
@@ -0,0 +1,167 @@
+package v1_test
```

Comment:
add more test cases for incorrect request body

Reply:
Done